finesse/hardware/plugins/stepper_motor/dummy.py

```diff
@@ -26,6 +33,8 @@ def __init__(
         """
         if steps_per_rotation < 1:
             raise ValueError("steps_per_rotation must be at least one")
+        if move_duration < 0.0:
+            raise ValueError("move_duration cannot be negative")
```

Comment:
I'd restrict to a positive number, as it should not be possible to have this duration as zero.
```suggestion
        if move_duration <= 0.0:
            raise ValueError("move_duration needs to be strictly positive")
```

Reply:
It is a *bit* weird, but it does sort of make sense to have it as an option for e.g. unit tests. We don't want to be waiting unnecessarily there. Instantaneous movement shouldn't actually break anything.